tests/functional/adapter/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+
+@pytest.fixture
```

Comment:
Curious what the advantage of having this be in the conftest vs a fixture defined in file?

Reply:
Any chance of there being a conflict with existing `conftest.py`? I'm guessing "no" since you got all green on the test suite. But, I'm not confident as to how pytest decides to scope `conftest.py` files.

Reply:
Re: @McKnight-42, I wanted this to be reusable for other tests. If we don't need to read project files for anything else, then it might not be appropriate to put it here. But I assumed that checking the parsed sql was something that might be more common in an adapter test.

Re: @VersusFacit, Closer to the test takes precedent. You only expose the fixture to tests that are in files while are alongside that particular conftest.py or in files that are in folders/subfolders alongside that particular conftest.py.

Reply:
Re: @McKnight-42 I completely misunderstood your comment. Yes, this can be in a separate fixtures file. There's no benefit to my knowledge. When I only have a few fixtures, I'll put them in conftest.py for simplicity. But once you hit a certain point, it's better to have it in a file. I could see us getting there, so we can start that here. I'll make the change.

Reply:
@VersusFacit I found this while researching the best way to structure this: https://docs.pytest.org/en/6.2.x/fixture.html#:~:text=described%20here.-,conftest,-.py%3A%20sharing%20fixtures. It verifies how the scope works.

Reply:
After going through everything, I decided to keep the fixture in `conftest.py`. There are multiple things that you can configure in the main `conftest.py`, but once you're in one level, the only purpose I can find for this file is to scope fixtures. In other words, this _is_ the fixtures file already.